### ISSUE TITLE
Plans to support cinder?

### ISSUE THREAD

**Issue by pgcorchary** [fr33jc/bang#19]
Are there plans to support cinder, with the goal of being able to build persistent volumes, attach them to instances and manage the connection (eg metadata notations). Thanks!

**Comment by fr33jc** [comment]
Yes indeed!  I have a branch to add EBS volumes now that I'll publish to bang-unstable.

**Comment by fr33jc** [comment]
@pgcorchary - Would you be interested in helping out with the OpenStack part?  I have limited access to OpenStack deployments.

**Comment by pgcorchary** [comment]
Well, 'Openstack' for me is HP Cloud, not 'vanilla' OS ...
